media-libs/libplacebo/libplacebo-2.72.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_REQ_USE="xml"
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://code.videolan.org/videolan/libplacebo.git"
+	inherit git-r3
+else
+	KEYWORDS="~amd64 ~ppc64 ~x86"
+	SRC_URI="https://code.videolan.org/videolan/libplacebo/-/archive/v${PV}/libplacebo-v${PV}.tar.gz"
+	S="${WORKDIR}/${PN}-v${PV}"
+fi
+
+inherit meson multilib-minimal python-any-r1
+
+DESCRIPTION="Reusable library for GPU-accelerated image processing primitives"
+HOMEPAGE="https://code.videolan.org/videolan/libplacebo"
+
+LICENSE="LGPL-2.1+"
+SLOT="0/$(ver_cut 2)" # libplacebo.so version
+IUSE="glslang lcms +opengl +shaderc test +vulkan"
+REQUIRED_USE="vulkan? ( || ( glslang shaderc ) )"
+RESTRICT="!test? ( test )"
+
+RDEPEND="glslang? ( dev-util/glslang[${MULTILIB_USEDEP}] )
+	lcms? ( media-libs/lcms:2[${MULTILIB_USEDEP}] )
+	opengl? ( media-libs/libepoxy[${MULTILIB_USEDEP}] )
+	shaderc? ( >=media-libs/shaderc-2017.2[${MULTILIB_USEDEP}] )
+	vulkan? ( media-libs/vulkan-loader[${MULTILIB_USEDEP}] )"
+DEPEND="${RDEPEND}"
+
+BDEPEND="virtual/pkgconfig
+	vulkan? (
+		${PYTHON_DEPS}
+		dev-util/vulkan-headers
```

Comment:
Per name of that package, I will guess that headers are used here, so `BDEPEND` is wrong. It must be in `DEPEND` instead.

Reply:
The reason I put it into BDEPEND here is because we specifically require the `vk.xml` it installs during build-time code generation. But looking at `man 5 ebuild` it seems that BDEPEND is only meant to be for executable tools, rather than "data files". So I'll change that.

That being said, in theory, the status quo up to this point has been to depend on the headers transitively via `media-libs/vulkan-loader`. I'm not sure if vulkan programs in general should have a separate dependency on `vulkan-headers` for the purposes of `#include <vulkan.h>`? i.e. should we move it to DEPEND or just omit it altogether?

Reply:
Yeah we don't do implicit deps, if it uses headers from vulkan-headers then this should be in DEPEND.